modules/core/src/test/java/org/locationtech/jts/geom/impl/PackedCoordinateSequenceTest.java

```diff
@@ -36,5 +41,103 @@ public PackedCoordinateSequenceTest(String name)
   CoordinateSequenceFactory getCSFactory() {
     return new PackedCoordinateSequenceFactory();
   }
+  
+  public void testDimensionAndMeasure()
+  {
+    CoordinateSequenceFactory factory = getCSFactory();
+    CoordinateSequence seq = factory.create(5, 2);
+    CoordinateSequence copy;
+    Coordinate coord;
+    Coordinate[] array;
+    
+    initProgression(seq);
```

Comment:
Is there an easy way to share this test code with the other class?

Reply:
Could place it in the superclass, the two tests are slightly different though. Since one store the coordinate instances (so == returns true) and one does not (so == returns false).

Reply:
Could have an override-able method around that check?